lib/beacon/content.ex

```diff
@@ -1667,6 +1680,108 @@ defmodule Beacon.Content do
     end
   end
 
+  # ERROR PAGES
+
+  @doc """
+  Returns an `%Ecto.Changeset{}` for tracking error page changes.
+
+  ## Example
+
+      iex> change_error_page(error_page, %{status: 404})
+      %Ecto.Changeset{data: %ErrorPage{}}
+
+  """
+  @doc type: :error_pages
+  @spec change_error_page(ErrorPage.t(), map()) :: Changeset.t()
+  def change_error_page(%ErrorPage{} = error_page, attrs \\ %{}) do
+    ErrorPage.changeset(error_page, attrs)
+  end
+
+  @doc """
+  Returns the error page for a given site and status code, or `nil` if no matching error page exists.
+  """
+  @doc type: :error_pages
+  @spec get_error_page(Site.t(), integer()) :: ErrorPage.t() | nil
+  def get_error_page(site, status) do
```

Comment:
Wdyt about being a bit more specific here?

```suggestion
  @type status_code :: 400..599

  @spec get_error_page(Site.t(), status_code()) :: ErrorPage.t() | nil
  def get_error_page(site, status) when status >= 400 and status <= 599 do
```

Reply:
Yeah in my first approach I was even more specific, with a list of only valid error codes (could later be used for form select).  I'll bring that back

Reply:
> Yeah in my first approach I was even more specific, with a list of only valid error codes (could later be used for form select). I'll bring that back

Yep that sounds good